### ISSUE TITLE
Bug: freeze when unmounting

### ISSUE THREAD

**Issue by RaptorZX4** [aldostools/webMAN-MOD#889]
For some reasons, when i unmount a game, sometimes it just freeze there and i can't do anything. I can still hold the power button or hold the PS button and select shut down the console, but even there, it gets stuck and the only way to really shutdown the console is to unplug it. And often it's going to ask me if i want to report the problem to Sony, and if i want to scan for problems.

I use Webman Mod 1.47.45, PS3Hen 3.2.2, on a PS3 Super Slim.

Hoping this is going to get fixed soon...

**Comment by aldostools** [comment]
Thank you for the report. Unless the bug can be reproduced on demand, it will be difficult to fix.

If you are using the dynamic fan controller, the freeze could be related to this feature which is known to cause issues on PS3HEN.

Another possibility is that the freeze could be related to the memory usage or the version of PS3HEN. If you're using the FPS plugin or another plugins, avoid using them to reduce the memory usage. Also try switching to the lite edition of webMAN MOD.

**Comment by RaptorZX4** [comment]
i don't think i use any other plugins (how can i see this?). I checked the Dynamic fan control, and Auto at: 68 Celcius, also checked Lowest: 25% - 80% fan speed. 
Because i want to keep temperatures a little lower while keeping the fan relatively quiet, the default "syscon" is kinda too low for my tastes, yes it's quieter but temperatures are kinda too high for my tastes.

Like i said it can't be reproduced on command. It only happen sometimes. So my guess is i have to do that often until i get the problem. 
Steps to reproduce usually: Select/mount a game from the WebMAN Games-Playstation 3 folder, then playing the game for a while, press PS button, exiting the game, and unmounting.

**Comment by aldostools** [comment]
My suggestion is that try using manual fan speed (e.g. 40% or any other value that give you a good balance between noise and temperature) and check if the freeze still happens with the same frequency.

**Comment by RaptorZX4** [comment]
> My suggestion is that try using manual fan speed (e.g. 40% or any other value that give you a good balance between noise and temperature) and check if the freeze still happens with the same frequency.

if i uncheck Dynamic fan control, and select Manual, it checks Dynamic fan control again.

**Comment by aldostools** [comment]
you have to save the settings after the change.

**Comment by RaptorZX4** [comment]
i did, restarted the console, and even there, if i select Manual, it auto-checks "Enable dynamic fan control"

if i uncheck it, it goes to "syscon" automatically.

**Comment by aldostools** [comment]
That's normal. "Enable dynamic fan control" checked means that webMAN will control the fan using auto, manual or auto#2 modes. If the option is unchecked, webMAN will not control the fan and SYSCON will be controlling it.

**Comment by RaptorZX4** [comment]
if i select syscon, then the Manual fan speed can't be selected, otherwise it's going to select Dynamic fan control.

Are you suggesting to use Dynamic fan control, but using Manual instead of Auto?

**Comment by aldostools** [comment]
Yes. I'm suggesting use Manual mode instead of Auto (Enable dynamic fan control will be checked).

**Comment by RaptorZX4** [comment]
alright i'll report if i still get freezes with Manual instead of "Auto at X Celcius"

**Comment by RaptorZX4** [comment]
Just a feedback, since i switched to Manual (fan set at 25%), temps seem to be a little bit lower than with the Auto setting. So that's a good thing so far.

**Comment by alfredox123** [comment]
Any of this tips worked for you for the freezes?

**Comment by RaptorZX4** [comment]
> Any of this tips worked for you for the freezes?

i'm using Manual at 25% now on my Super Slim PS3, and temps are a little bit lower since then. It may be a little bit more noisy but i prefer that than having the console having super hot temperatures, so it's kind of a good compromise at 25% fan speed. 
I also repasted the console a year ago, just to make sure it won't die on me.

**Comment by alfredox123** [comment]
ok but the freezes on the console are gone when using manual fan speed?

**Comment by RaptorZX4** [comment]
> ok but the freezes on the console are gone when using manual fan speed?

it seem to be the case, i never had a freeze since then.
